INSTALL.rst

```diff
@@ -1,123 +1,104 @@
+
+venv
+====
+When using ``venv``, you can get started with the following commands::
+
+  # Take note of your current directory
+  pwd
+  # Create a virtualenv named ``skimage`` that lives in the directory of
+  # the same name
+  python -m venv skimage
+  # Activate it
+  source skimage/bin/activate
+  # Install a precompiled scikit-image wheel from PyPi
   pip install scikit-image
+  # Install all other packages you need
+  pip install ...
 
-If you are running Anaconda or miniconda, use::
-
-  conda install -c conda-forge scikit-image
-
-The wheels can be downloaded manually from `PyPI <https://pypi.org/project/scikit-image/#files>`__.
-
-2. Development Installation:
-----------------------------
-
-You can install the ``scikit-image`` development version if either your
-distribution ships an outdated version or you want to develop and work on new
-features before the package is released officially.
-
-First, uninstall any existing installations::
-
-  pip uninstall scikit-image
-
-or, on conda-based systems::
-
-  conda uninstall scikit-image
-
-Now, clone scikit-image on your local computer, and install::
-
-  git clone https://github.com/scikit-image/scikit-image.git
-  cd scikit-image
-  pip install -e .
-
-To update the installation::
-
-  git pull  # Grab latest source
-  pip install -e .  # Reinstall
-
-Platform-specific notes follow below.
-
-a. Windows
-``````````
-If you experience the error ``Error:unable to find vcvarsall.bat`` it means
-that your computer does not have recommended compilers for Python. You can
-either download and install Windows compilers from `here`_  or use
-`MinGW compilers`_ . If using `MinGW`, make sure to correctly configure
-``distutils`` by modifying (or create, if not existing) the configuration file
-``distutils.cfg`` (located for example at
-``C:\Python26\Lib\distutils\distutils.cfg``) to contain::
+Now, whenever you want to use this specific version of scikit-image, make sure
+you activate it by::
 
-  [build]
-   compiler=mingw32
+  # Change to the directory where you created the virtual environment
+  cd YOUR_DIRECTORY
+  # Activate it
+  source skimage/bin/activate
+  # Use python as you wish
 
-A run-through of the compilation process for Windows is included in
-our `setup of Azure Pipelines`_ (a continuous integration service).
 
-.. _miniconda: http://conda.pydata.org/miniconda.html
-.. _python.org: http://python.org/
-.. _setup of Azure Pipelines: https://github.com/scikit-image/scikit-image/blob/master/azure-pipelines.yml
-.. _here: https://wiki.python.org/moin/WindowsCompilers#Microsoft_Visual_C.2B-.2B-_14.0_standalone:_Visual_C.2B-.2B-_Build_Tools_2015_.28x86.2C_x64.2C_ARM.29
-.. _venv: https://docs.python.org/3/library/venv.html
-.. _virtual environments: https://docs.python-guide.org/en/latest/dev/virtualenvs/
-.. _MinGW compilers: http://www.mingw.org/wiki/howto_install_the_mingw_gcc_compiler_suite
+Should you need to have access to the wheels offline, they can be downloaded
```

Comment:
I'd write either _Should you need to access the wheels offline_ or _If you need to have access to the wheels offline_, preferring the second. Maybe even _If you need to access the wheels offline, download them at `<URL>`_. Thoughts?